src/common/preferences/preferences.html

```diff
@@ -104,6 +102,19 @@
 				</div>
 			</div>
 			<div class="group">
+				<div class="group-title">Report Errors</div>
+				<div class="group-content">
+					<div id="general-no-errors">
+						Zotero Connector has not experienced any errors since it was first started.
```

Comment:
Yea I've just been moving this around for UI reasons, but fair point. Might want to revise more copy in the connector.